files/default/neutron-ha-tool.py

```diff
@@ -81,6 +81,9 @@ def parse_args():
                     help='Replicate DHCP configuration to all agents')
     ap.add_argument('--now', action='store_true', default=False,
                     help='Migrate Routers immediately without a delay.')
+    ap.add_argument('--wait-for-ports', action='store_true', default=False,
```

Comment:
Hm, I not really sure yet what's the best default here. So I kept backwards compatible. 

Does it really make sense in the HA fail-over use case (when a single node, that is most likely offline already, needs to be evacuated quickly) to add all the waiting? I'd think that in that case it's preferable to have this script finish as fast as possible.  And after all this is still the default use-case of neutron-ha-tool.